src/game.cpp

```diff
@@ -2166,6 +2170,10 @@ bool Game::connectToServer(const std::string &playername,
 			}
 
 			// Break conditions
+			if (*aborted) {
```

Comment:
Needs a better name. Perhaps `abort_connecting`?

Reply:
IDK, I just use what's already there. Should we change only for this or also other?

Reply:
Only this function, as it was touched. `connect_aborted` seems to be the better choice (same name used [here](https://github.com/srifqi/minetest/blob/fe7254ea07c6cc9849adf18723acd1addf595ec0/src/game.cpp#L1909))